src/renderer/helpers/api/local.js

```diff
@@ -0,0 +1,19 @@
+import { Innertube } from 'youtubei.js'
+
+let searchSuggestionsInnerTube = null
+
+export async function getYTSearchSuggestions(query) {
```

Comment:
What are you planning on naming the invidious function once it moves to a helper? Just want to make sure the names don't collide. Other than that, code looks good to me (haven't tested yet)

Reply:
All the Invidious ones will have Invidious in the name. It probably makes sense to use local instead of YouTube in the name, so I'll rename this one as well.

Reply:
Sounds good